examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -0,0 +1,274 @@
+##############################################################################
+# Note that the "WAGE" distribution has a long tail and we could take its log
+# to simplify our problem getting closer to a normal distribution.
+# For all 3 variables, "EDUCATION", "EXPERIENCE", and "AGE", the "WAGE" is
+# increasing when these variables are increasing. Also, the "EXPERIENCE" and
+# "AGE" are correlated.
+#
+# The pipeline
+# ............
+
+survey.data.info()
+
+#############################################################################
+# The dataset is composed by columns with different data types and we need to
+# apply a specific preprocessing for each data types.
+# Our pre-processor will
+#
+# - one-hot encode (i.e., generate a column by category) the categorical
+#   columns;
+# - replace by 0 and 1 the categories of binary columns;
+# - keep numerical values as they are.
+
+from sklearn.compose import make_column_transformer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.preprocessing import OrdinalEncoder
+
+categorical_columns = ['RACE', 'OCCUPATION', 'SECTOR']
+binary_columns = ['MARR', 'UNION', 'SEX', 'SOUTH']
+numerical_columns = ['EDUCATION', 'EXPERIENCE', 'AGE']
+
+preprocessor = make_column_transformer(
+    (OneHotEncoder(), categorical_columns),
+    (OrdinalEncoder(), binary_columns),
+    remainder='passthrough'
+)
+
+##############################################################################
+# To describe the dataset as a linear model we choose to use a ridge regressor
+# and to model le log of the "WAGE".
+
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import RidgeCV
+from sklearn.compose import TransformedTargetRegressor
+
+model = make_pipeline(
+    preprocessor,
+    TransformedTargetRegressor(
+        regressor=RidgeCV(),
+        func=np.log10,
+        inverse_func=sp.special.exp10
+    )
+)
+
+##############################################################################
+# Processing the dataset
+# ......................
+#
+# First of all we split the sample in a train and a test dataset.
+
+from sklearn.model_selection import train_test_split
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, random_state=42
+)
+
+##############################################################################
+# Then we fit the model
+
+model.fit(X_train, y_train)
+
+##############################################################################
+# We can check the performance of the computed model using, for example, the
+# median absolute error of the model.
+
+from sklearn.metrics import median_absolute_error
+
+y_pred = model.predict(X_train)
+mae = median_absolute_error(y_train, y_pred)
+string_score = 'MAE on training set: {0:.2f} $/hour'.format(mae)
+y_pred = model.predict(X_test)
+mae = median_absolute_error(y_test, y_pred)
+
+string_score += '\nMAE on testing set: {0:.2f} $/hour'.format(mae)
+
+fig, ax = plt.subplots(figsize=(6, 6))
+sns.regplot(y_test, y_pred)
```

Comment:
there should be a way to have cross-refs with sphinx for seaborn functions as well?

Reply:
`intersphinx_mapping` for seaborn added in `conf.py`